std/uuid/v4.ts

```diff
@@ -1,21 +1,19 @@
 // Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
 import { bytesToUuid } from "./_common.ts";
 
-const UUID_RE = new RegExp(
-  "^[0-9a-f]{8}-[0-9a-f]{4}-4[0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$",
-  "i",
-);
+const UUID_RE =
+  /^[0-9a-f]{8}-[0-9a-f]{4}-4[0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$/i;
 
 /**
- * Validates the UUID v4
- * @param id UUID value
+ * Validates the UUID v4.
+ * @param id UUID value.
  */
-export function validate(id: string): boolean {
```

Comment:
Why remove the return value type?

Reply:
It's implicit.  TypeScript can infer these typings.

Reply:
Our doc generator can not though, so please re-add the explicit types.

Reply:
@lucacasonato That's probably something that we should fix, then.  Deno brags its seamless TS integration, so it would be a shame if people had to be over-explicit in their TS to just get documentation generated.

Reply:
Unrelated to this PR though, so please revert. Even if this were not a concern we prefer explicit return types, because it exposes errors related to missing/incorrect return statements in branches in a more logical way, and makes the code more readable for people who do not use a TypeScript integration in their editor.

Reply:
> We should really add this to the style guide (we used to have a lint for this).

Was just typing a question if I should post a PR later tonight.

> Unrelated to this PR though, so please revert. Even if this were not a concern we prefer explicit return types, because it exposes errors related to missing/incorrect return statements in branches in a more logical way, and makes the code more readable for people who do not use a TypeScript integration in their editor.

Got it, I'll adjust the code after I finish my homework in 'bout two hours.